notebooks/00_core.ipynb

```diff
@@ -30,6 +30,7 @@
     "import os\n",
```

Comment:
<ul><li>i would call this <code>make_polynomial</code>  or <code>make_polynomial_data</code> </li><li>i would replace <code>w</code>  with <code>weights</code>  </li><li>i would replace <code>y_data</code>  with <code>y</code>  for consistency with the docstring</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lewtun/dslectures/pull/32/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='52'/>